test/i18next_test.dart

```diff
@@ -198,6 +210,58 @@ void main() {
     });
   });
 
+  group('fallback', (){
+    test('given a global fallback key substitution', () {
+      const fallbackNamespace1 = 'fallback_namespace_1';
+      i18next = I18Next(
+        locale,
+        resourceStore,
+        options: const I18NextOptions(
+          fallbackNamespaces: [fallbackNamespace1],
+        ),
+      );
+
+      mockKey('key', 'fallbackValue', ns: fallbackNamespace1);
+      mockKey('key', 'value', ns: namespace);
+
+      expect(i18next.t('key'), 'fallbackValue');
+      expect(i18next.t('$namespace:key'), 'value');
+    });
+
+    group('given 2 global fallback keys subsitution',(){
+      const fallbackNamespace1 = 'fallback_namespace_1';
+      const fallbackNamespace2 = 'fallback_namespace_2';
+
+      setUp(() {
+        i18next = I18Next(
+          locale,
+          resourceStore,
+          options: const I18NextOptions(
+            fallbackNamespaces: [fallbackNamespace1, fallbackNamespace2],
+          ),
+        );
+      });
+
+      test('key only exists in second fallbackNamespace', (){
+        mockKey('key2', 'fallbackValue2', ns: fallbackNamespace2);
+        mockKey('key2', 'value2', ns: namespace);
+
+        expect(i18next.t('key2'), 'fallbackValue2');
+        expect(i18next.t('$namespace:key2'), 'value2');
+      });
+
+      test('key exists in both first and second fallbackNamespace', (){
+        mockKey('key', 'fallbackValue', ns: fallbackNamespace1);
+        mockKey('key', 'fallbackValue', ns: fallbackNamespace2);
+        mockKey('key', 'value', ns: namespace);
+
+        expect(i18next.t('key'), 'fallbackValue');
```

Comment:
Need to differentiate the `fallbackValue1` for `fallbackNamespace1` 
and `fallbackValue2` for `fallbackNamespace2`

otherwise, the test isn't explicitly covering which fallback it is falling to (in this case, fallbackNamespace1)